project/EnsimeBuild.scala

```diff
@@ -58,6 +58,7 @@ object EnsimeBuild extends Build with JdkResolver {
 
   lazy val commonSettings = scalariformSettings ++ basicSettings ++ Seq(
     //resolvers += Resolver.sonatypeRepo("snapshots"),
+    resolvers += "Spring IO" at "http://repo.spring.io/libs-release-remote",
```

Comment:
they don't, they use `sbt` to download ensime-server (and all its dependencies). If we have dependencies that aren't on maven central, that repo needs to be added to the bootstrapper